src/routes/docs/references/[version]/models/[model]/+page.server.ts

```diff
@@ -59,13 +63,82 @@ export const load: PageServerLoad = async ({ params }) => {
         })
     };
 
-    const example = model.properties.reduce<Record<string, unknown>>((carry, property) => {
-        carry[property.name] = property.example;
+    const example = generateExample(schema, api);
 
-        return carry;
-    }, {});
     return {
         model,
         example
     };
 };
+
+const generateExample = (schema: OpenAPIV3.SchemaObject, api: OpenAPIV3.Document<{}>): Object => {
+
+    const properties = Object.keys(schema.properties ?? {}).map((key) =>{
+        const name = key;
+        const fields = schema.properties?.[key]; 
+        return {
+            name,
+            ...fields
```

Comment:
can we move that part into the specs module?

Reply:
Done